client/src/TitleScreen/TitleScreen.js

```diff
@@ -122,6 +122,13 @@ export default class SplashScreen {
 		document.body.appendChild(this.HTML);
 		setTimeout(() => {
 			this.HTML.style.opacity = '1';
+			document
+					.querySelector('.SplashScreen-container .icons')
+					.addEventListener(
+						this.muteToggle = false,
+						'click',
+						mutePage()
```

Comment:
`muteToggle` does not have to be bound to the class. Declare it in the function body of the callback you pass into the event listener.

```suggestion
						'click',
						() => {
                    let muteToggle = true
                    mutePage(muteToggle)
                    muteToggle = !muteToggle 
						}
```